receiver/hostmetricsreceiver/go.mod

```diff
@@ -8,6 +8,7 @@ require (
 	github.com/open-telemetry/opentelemetry-collector-contrib/internal/coreinternal v0.86.1-0.20231006161201-d364ad61c4d7
 	github.com/open-telemetry/opentelemetry-collector-contrib/internal/filter v0.86.1-0.20231006161201-d364ad61c4d7
 	github.com/open-telemetry/opentelemetry-collector-contrib/pkg/pdatatest v0.86.1-0.20231006161201-d364ad61c4d7
+	github.com/prometheus/procfs v0.11.1
```

Comment:
Maybe a stupid q, but just to clarify we why did do we use `procfs` over `gopsutil`? I have something against `gopsutil` in mind regarding the current windows frequency request?

Reply:
[`gopsutil` ](https://github.com/shirou/gopsutil/blob/2fabf15a16dca198f735a5de2722158576e986a9/cpu/cpu_linux.go#L145-L148)overrides the value coming from `/proc/cpuinfo` with the value from `/cpufreq/cpuinfo_max_freq` so as to be aligned with Windows behavior.

See also https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/26532#issuecomment-1731163828.

Hence we decide that it's better to provide the current frequency instead of max. `procfs` reports this directly in Linux so it's our best option here.

Reply:
Thanks for clarification!